src/main/java/co/chlg/javaimpdec/shell/LambdaCommands.java

```diff
@@ -28,9 +34,11 @@ private int doUpdateVocalCount(@ShellOption List<String> words) {
 
   @ShellMethod(group = "lambda", value = "Ejercicio de sincronia y lambda")
   private List<Boolean> doAllowEntry(@ShellOption List<String> party) {
-    Supplier<Boolean> releaseAndNull = null;
     // TODO: Can implement releaseAndNull supplier to help map easily in the implementation
-    return null;
+    Semaphore semaforo = new Semaphore(2);
```

Comment:
This needs to be a class field and also the method can be improved using a `Stream<String>`